app/templates/service.html

```diff
@@ -32,13 +32,22 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     {% if service_unavailability_information.type == 'framework_update' %}
+    {# TODO: Hardcoding this banner for G11 for now, should be derived from the framework object  #}
+       {% if service.frameworkName == 'G-Cloud 11' %}
+           {{ dmBanner({
+        "title": "This G-Cloud 11 service has been suspended and is no longer searchable on the Digital Marketplace.",
+        "text": "The G-Cloud 11 framework expires on 18 December 2020 to allow in progress procurements to be concluded. Existing contracts with {} are still valid.".format(
```

Comment:
```suggestion
        "text": "The G-Cloud 11 framework expires on 18 December 2020 to allow procurements in progress to be concluded. Existing contracts with {} are still valid.".format(
```

Reply:
Or should we be more assertive? "Procurements in progress to be concluded before the G-Cloud 11 framework expires on 18 December 2020."?

Reply:
I feel like the wording is really one for @NoraGDS who's not in today. Can we use this for now and then can revise when @NoraGDS is back?

Reply:
who is the source for this content? I agree with the change in any case. I'd still like Nora's eyes on this but I am happy to go ahead for now and change later if needed.

Reply:
Oh, I see this wording came from CCS. Never mind, then. Happy for this content to go in as-is and let a content professional to take a look tomorrow.

Reply:
yes, source is CCS verbatim for now.